### PR TITLE
Add Sponsorship Logos

### PR DESCRIPTION
**Pull request by El-Mathematica**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,5 +32,8 @@
 <span class="delay d20">    <i c=comment><span aria-hidden=true>/*</span>                                          <span aria-hidden=true>*/</span></i>    </span>
 <span class="delay d21">    <i c=comment><span aria-hidden=true>/*</span>  <a href=/join>Join CSS now through Guild of Students</a>  <span aria-hidden=true>*/</span></i>    </span>
     </pre>
-  </div>
-</div>
+        </div>
+        <img src="cssuob.github.io\assets\images\logos\american_express_logo.png" alt="American Express Logo">
+        <img src="cssuob.github.io\assets\images\logos\kainos_logo.webp" alt="Kainos Logo">
+        <img src="cssuob.github.io\assets\images\logos\university_of_birmingham_logo.png" alt="University of Birmingham Logo">
```
Comment (JTWWilson) [inline index.html:38]: Not pointing properly to the images, change to just /assets/images/logos/{NAME}

File: index.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
...
 </abbr>
 <span>                                                       </span>
-<span class="delay d13">    <i c=comment itemscope itemtype=http://schema.org/Organization><span aria-hidden=true>/*</span>         <h1 itemprop=name>Computer Science Society</h1>         <span aria-hidden=true>*/</span></i>    </span>
+<span class="delay d13">    <i c=comment itemscope itemtype=http://schema.org/Organization><span aria-hidden=true>/*</span>         <h1 itemprop= name>Computer Science Society</h1>         <span aria-hidden=true>*/</span></i>    </span>
```
Comment (JTWWilson) [inline index.html:25]: unnecessary

**Review by JTWWilson — CHANGES_REQUESTED** [review]

**Comment by AlexJBanks** [comment]
Screenshot_20211109-210545
On mobile, first image renders too low on the screen. Rest are not on screen

**Comment by AlexJBanks** [comment]
superseded by #218

**Pull request closed without merging**